Tests/SentryTests/SentrySpanContextTests.swift

```diff
@@ -60,6 +60,32 @@ class SentrySpanContextTests: XCTestCase {
         XCTAssertEqual(data["status"] as? String, "ok")
     }
     
+    func testSampledNoSerialization() {
+        let id = SentryId()
+        let spanId = SpanId()
+        let parentId = SpanId()
+        
+        let spanContext = SpanContext(trace: id, spanId: spanId, parentId: parentId, operation: someOperation, sampled: .no)
+        spanContext.status = .ok
```

Comment:
I see a grand potential of removing those duplications across tests and creating an aux methox that takes param to build the span/spanContext

Reply:
I agree with this statement. Thanks for pointing it out @marandaneto.

Reply:
Can we postpone it to a "test fixing" PR?